tests/TranslatableTest.php

```diff
@@ -8,227 +8,124 @@
 
 class TranslatableTest extends TestCase
 {
+    public $post;
+
+    protected function getEnvironmentSetUp($app)
+    {
+        $app['config']->set('app.locale', 'en');
+        $app['config']->set('database.default', 'testbench');
+        $app['config']->set('database.connections.testbench', [
+            'driver'   => 'sqlite',
+            'database' => ':memory:',
+            'prefix'   => '',
+        ]);
+    }
+    
     public function setUp(): void
     {
         parent::setUp();
         $this->loadMigrationsFrom(__DIR__ . '/migrations');
+
+        // POST MODEL
+        $this->post = Post::find(1);
+
+        if (!$this->post) {
+            $this->post = Post::create(['published' => true]);
+            PostTranslation::create([
+                'locale' => 'en',
+                'post_id' => 1,
+                'title' => 'post_test_title',
+                'content' => 'post_test_content',
+                'description' => 'post_test_description',
+                'keywords' => 'post_test_keyword',
+            ]);
+
+
+            Post::create(['published' => true]);
+            PostTranslation::create([
+                'locale' => 'en',
+                'post_id' => 2,
+                'title' => 'post_2_test_title',
+                'content' => 'post_2_test_content',
+                'description' => 'post_2_test_description',
+                'keywords' => 'post_2_test_keyword',
+            ]);
+        }
     }
 
     public function testTranslate()
     {
-        PostTranslation::create([
-            'locale' => 'en',
-            'post_id' => 1,
-            'title' => 'post_test_title',
-            'content' => 'post_test_content',
-            'description' => 'post_test_description',
-            'keywords' => 'post_test_keyword',
-        ]);
-        $post = Post::create([
-            'id' => 1,
-            'published'=> true,
-        ]);
-
-        $actual = $post->translate('en')->title;
+        $this->post->translate('en')->title = 'post_test_title';
```

Comment:
Do you forget to call the `$this->post->save()` method?

Reply:
yes i think so